apps/condo/domains/billing/schema/BillingIntegrationOrganizationContext.js

```diff
@@ -86,6 +90,25 @@ const BillingIntegrationOrganizationContext = new GQLListSchema('BillingIntegrat
         delete: false,
         auth: true,
     },
+    hooks: {
+        validateInput: async ({ operation, resolvedData, addValidationError }) => {
+            // should have only one explicit (hidden = false) context in organization!
+            if (operation === 'create') {
+                const integration = await getById('BillingIntegration', get(resolvedData, ['integration']))
+                if (get(integration, 'isHidden') === true) { return } // we can add as many b2c integrations as we like
```

Comment:
it's better to extract all this staff in a function with clear name

Reply:
you mean -- get integration?